cmd/backup-manager/app/backup/manager.go

```diff
@@ -582,6 +616,37 @@ func (bm *Manager) stopLogBackup(ctx context.Context, backup *v1alpha1.Backup) (
 	return updateStatus, "", nil
 }
 
+// pauseLogBackup stops log backup.
+func (bm *Manager) pauseLogBackup(ctx context.Context, backup *v1alpha1.Backup) (*controller.BackupUpdateStatus, string, error) {
+	started := time.Now()
+
+	// change Prepare to Running before real backup process start
+	if err := bm.StatusUpdater.Update(backup, &v1alpha1.BackupCondition{
+		Command: v1alpha1.LogPauseCommand,
+		Type:    v1alpha1.BackupRunning,
```

Comment:
Should we need to set `Paused` here?

Reply:
The running state here means the `command` is running rather than the `log backup` is running.